src/polhemus_ros_driver/knuckle_position_calibration_action_server.py

```diff
@@ -19,17 +19,34 @@
 from enum import Enum
 
 
-def map_range(value, in_min, in_max, out_min, out_max):
+def map_range(input, in_min, in_max, out_min, out_max):
+    """
+        Returns the convereted value of 'input' from range [out_min, out_max] into range [in_min, in_max]
+        @param value: Value to be mapped
+        @param in_min: Minimal value of 'input'
+        @param in_max: Maximal value of 'input'
+        @param out_min: Minimal value of mapped output
+        @param in_max: Maximal value of mapped output
+    """
     return (value - in_min) * (out_max - out_min) / (in_max - in_min) + out_min
 
 
 def calculate_distance(point1, point2):
+    """
+        Returns the distance between two points.
+        @param point1: Second point
```

Comment:
Should it be like this?
```suggestion
        @param point1: First point
```

Reply:
Also maybe worth saying the format you are expecting points to be given